doc/en/user/source/production/container.rst

```diff
@@ -5,6 +5,11 @@ Container Considerations
 
 Java web containers such as `Tomcat <http://tomcat.apache.org>`_ or `Jetty <http://www.mortbay.org/jetty/>`_ ship with configurations that allow for fast startup, but don't always deliver the best performance.
 
+Use a supported Java version
+----------------------------
+
+GeoServer is built and tested with Java 8, `supported up to 2026 <https://adoptopenjdk.net/support.html#roadmap>`_, and Java 11, `supported up to 2024 <https://adoptopenjdk.net/support.html#roadmap>`_. More recent releases of Java are supported for only six months and contain experimental features, they are not tested with GeoServer, thus not recommended for production: the application might not start or misbehave, and the recommended JVM flags might no longer be applicable.
+
```

Comment:
We already have a place for this in our user manual (literally the previous page) and I would like to cross link there, rather than duplicate suggestions.

Introducing a heading "Running on Java 8" there may be more clear?

Reply:
@jodygarnett works for me. Can you amend this PR accordingly? 
The PR was triggered by the -XX:+UseStringDeduplicationJVM option not working anymore in recent versions of GeoServer. What's your take on it? Just remove it?

Reply:
I was going to ask you to just close this pull request, the previous page [Java Considerat ions](https://docs.geoserver.org/latest/en/user/production/java.html) talks about using a supported java version very clearly.

We also have lots of warnings in the [installation](https://docs.geoserver.org/latest/en/user/installation/index.html#installation) pages on use of Java 8 or Java 11.

![image](https://user-images.githubusercontent.com/629681/88006977-c6247b80-cac1-11ea-8a04-2b5abf3e1c49.png)

Reply:
Roger that, closing.